lesson06/assignment/Accordian.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```

Comment:
Reminder: HTML and CSS naming convention is "kebab case", all lower case, with hyphens instead of spaces. So this file should be `accordian.html`.